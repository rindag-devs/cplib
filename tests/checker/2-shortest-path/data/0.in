4 6
1 2 2
2 3 2
2 4 1
1 3 1
3 4 3
1 4 4
