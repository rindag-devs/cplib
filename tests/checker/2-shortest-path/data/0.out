4
3 1 3 4
